docs/MERGE-WORKFLOW.md

```
@@ -0,0 +1,164 @@
+# Merge Workflow
+
+Panopticon's merge workflow is a four-state pipeline between two actors: the
+**dashboard server** and **GitHub**. Agents participate in the upstream work
+(implementation, review, test) but no agent makes the merge decision and no
+agent performs the rebase.
+
+## State Machine
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)